services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/PushStatusCollector.java

```diff
@@ -150,6 +151,7 @@ private void scanDaVinciPushStatus() {
       }
       ExecutionStatusWithDetails daVinciStatus = pushStatus.getDaVinciStatus();
       if (daVinciStatus.isNoDaVinciStatusReport()) {
+        LOGGER.info("Received empty DaVinci status report for topic: {}", pushStatus.topicName);
```

Comment:
what if server ingested too quickly before the dvc poll and by next poll DVC also finished and since it stopped emitting heartbeat, then it can return empty report ?

Reply:
If there are some staled instance report by checking HB timestamp, these instances' status will be ignored, and another config will limit the maximum number of retries and the job will succeed after retries.